rocket/hello-world/README.md

```diff
@@ -0,0 +1,7 @@
+# Hello World Application using Rocket
+This example shows how to launch a hello world app using the [rocket](https://rocket.rs/v0.4/guide/introduction/) framework and the `#[shuttle_service::main]` attribute.
```

Comment:
The current version of rocket is 0.5.0-rc.2, but we can just link to their homepage.
```suggestion
This example shows how to launch a hello world app using [rocket](https://rocket.rs) and shuttle.
```